lxd/instance/drivers/vm_qemu.go

```diff
@@ -72,10 +72,10 @@ func Load(s *state.State, args db.InstanceArgs, profiles []api.Profile) (instanc
 	return vm, nil
 }
 
-// Instantiate creates a Qemu struct without expanding config. The expandedDevices argument is
+// qemuInstantiate creates a Qemu struct without expanding config. The expandedDevices argument is
 // used during device config validation when the devices have already been expanded and we do not
 // have access to the profiles used to do it. This can be safely passed as nil if not required.
-func Instantiate(s *state.State, args db.InstanceArgs, expandedDevices deviceConfig.Devices) *Qemu {
+func qemuInstantiate(s *state.State, args db.InstanceArgs, expandedDevices deviceConfig.Devices) *Qemu {
```

Comment:
@stgraber now that qemu and lxc implementations will share the same package, shall I rename the `Qemu` type to `vmQemu` to match `containerLXC` when it gets moved in in the future? This will also better match the file name `vm_qemu.go`.

Reply:
No, I think we should just rename containerLXC to LXC. There are no non-vm Qemu and there are no non-container LXC, so having that in the name isn't really useful.